src/commentPage.js

```diff
@@ -0,0 +1,23 @@
+import { getItemComments } from './involvementAPI';
```

Comment:
- [ ] I understand using two different pop-ups could be hard to do. On this project, is important to use it because is part of the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=The%20comments%20popup.,groups%20of%203%20students)). Please, use pop-up for comments too, this is in place to open a new page. 🐶

Reply:
* Done 👍